src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/tests/Radio.test.js

```diff
@@ -1,25 +1,25 @@
 // @flow
-import {shallow, render} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
```

Comment:
would be nice to use the `@testing-library/user-event` package instead of `fireEvent` 🙂 see https://github.com/sulu/sulu/pull/6783 for some examples